docs/docs/getting_started/quick_start.md

```diff
@@ -110,7 +111,7 @@ The proof is now generated in the `target` folder. To verify it we first need to
 bb write_vk -b ./target/hello_world.json -o ./target
 
 # Verify the proof
-bb verify -k ./target/vk -p ./target/proof
+bb verify -k ./target/vk -p ./target/proof -i ./target/public_inputs
```

Comment:
This worked for me on beta.4 and bb 0.84 without passing the `-i ./target/public_inputs`. Is this just the recommended way to do it now? Or will it be required in the future?

Reply:
This value is the default path so it will work even if you don't pass it explicitly.